data/domain/cards.tsx

```diff
@@ -148,6 +164,29 @@ export const updateCards = (data: Map<string, any>) => {
     const players = data.get("players") as Player[];
     const optLacc = data.get("OptLacc");
 
+    if (rift.bonuses.find(bonus => bonus.name == "Skill Mastery")?.active) {
+
+        const skillMastery = rift.bonuses.find(bonus => bonus.name == "Skill Mastery") as SkillMastery;
+
+        Object.entries(skillMastery.skillLevels).map(([skillAsString, skillLevel], index) => {
+            const skillName = SkillsIndex[parseInt(skillAsString)];
+            const skillIndex = SkillsIndex[skillName as keyof typeof SkillsIndex];
+
+            if (skillMastery.getSkillRank(skillIndex) >= 3 && SkillsforIDCardPassiveBonus[skillIndex].length > 0) {
+                cards.forEach(card => {
```

Comment:
minor: this is personal preference but this feels reversed. 

i.e. instead of for every card search the list, I'd say for every id in the list, find the matching card .. if that makes sense.

Again personal approach, not sure it matters long run haha

Reply:
I think it's more efficient indeed, less loops for nothing, gonna do it that way